java/src/jmri/jmrit/display/CoordinateEdit.java

```diff
@@ -332,7 +332,7 @@ public void actionPerformed(java.awt.event.ActionEvent e) {
     public void initSetLevel() {
         oldX = pl.getDisplayLevel();
         textX = new javax.swing.JLabel();
-        textX.setText("level= " + pl.getDisplayLevel());
+        textX.setText(Bundle.getMessage("Level") + " = " + pl.getDisplayLevel());
```

Comment:
> Is there any place in JMRI where we already apply MassageFormat?

There are properties strings with '{..}' in e.g. java/src//jmri/jmrix/JmrixBundle.properties, java/src//jmri/jmrit/Bundle.properties, java/src//jmri/implementation/ImplementationBundle.properties, even java/src//jmri/NamedBeanBundle.properties.  You can pick one of those and search the code to find how it's used.  For example, from NamedBeanBundle.properties, 
````
JavaVersionCredit = Java version {0} ({1})
````
is used in java/src//jmri/swing/AboutDialog.java:
````
        pane1.add(new JLabel(Bundle.getMessage("JavaVersionCredit",
                System.getProperty("java.version", "<unknown>"),
                Locale.getDefault().toString())));
````
That one uses the support built into Bundle.getMessage to insert two strings (2nd two lines) into the message.